app/uk/gov/hmrc/tariffclassificationfrontend/views/not_authorized.scala.html

```diff
@@ -21,14 +21,8 @@
 
 @main(bodyClasses = None) {
 
-    <h1 class="heading-large">You are not authorized to access this page.</h1>
+    <h1 class="heading-large">You are not authorised to access this page.</h1>
```

Comment:
British English

Reply:
"Authorised" is British English, the `z` is US style

Reply:
I know!